README.md

```
@@ -1,159 +1,206 @@
-    "metadata": {
-      "customer_source": "mailing_list"
-    },
-    "currency": null
+npm run build
 ```
 
-### STEP4: Launch React application
+## Project Structure
 
-In the `front` directory, we can run React application to visualize the customer data that get from the API.
+```
```

markdownlint-cli2 (0.20.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)